engine/classes/Elgg/Router.php

```diff
@@ -56,7 +56,7 @@ public function route(Request $request) {
 		$is_walled_garden = _elgg_services()->config->get('walled_garden');
 		$is_logged_in = _elgg_services()->session->isLoggedIn();
 		$url = elgg_normalize_url($identifier . '/' . implode('/', $segments));
-		
```

Comment:
Yeah, I did a soft reset against master that wasn't in sync. I will fix that later.